src/data/graphql/queries.graphql

```diff
@@ -650,4 +650,23 @@ query SubgraphActions($colonyAddress: String!) {
       }
     }
   }
+  events(where: { associatedColony_contains: $colonyAddress, name_in: ["TokensMinted(address,address,uint256)"] }) {
+    id
+    transaction {
+      hash: id
+      block {
+        id
+        timestamp
+      }
+    }
+    associatedColony {
+      token {
+        decimals
+        symbol
+      }
+    }
+    args
+    address
```

Comment:
This is the `agent`, right ?

Maybe rename it so it's a bit more clear

Reply:
It's the address of the contract that emitted the event, agent is sitting in args object. i will add comment there